### DISCUSSION TITLE
Wow.. and thanks

### DISCUSSION THREAD

**Discussion by gnoshme** [magicbug/Cloudlog#1965] [General]
I've been banging my head against a brick wall and not logging at all outside FT8 because as a linux user, and someone who doesn't like software that looks like it was built in 1998, I've felt really stuck.  Was about to roll up my sleeves and make a very basic Rails app.. and then I found this.  Great work.  Does exactly what I need it to do.

Took me a couple of hours to setup a VM on Contabo (cheap) and get it up and running.  Really excited to make voice contacts now.

Thanks. 

Keifer / K3IFR

**Comment by magicbug** [comment]
HI, @gnoshme thanks for the kind words, that's exactly why I built it, got so annoyed with the old style software available

**Comment by gnoshme** [comment]
Yeah. Seriously.  In the space of an afternoon I setup:

- a lamp stack on a new vm
- installed cloudlog
- got qrz lookups working
- imported qrz log
- got real time logging for voice into qrz working
- got FT8 contacts successfully logging to cloudlog via GridTracker on connected pi
- got the radio frequency successfully pushed to Cloudlog using fldig and cloudlogbashcat on connected pi

This is everything I wanted.  Going to tackle LOTW when after they accept my first born child and give me a login.

Keep up the good work. Will contribute where I can and expect a donation incoming.

Keifer / K3IFR

**Comment by magicbug** [comment]
LoTW should be easy compared to the hoops us from outside the US deal with :-)
